test/content/support.js

```diff
@@ -241,4 +241,17 @@ function resetDB() {
 	}).then(function() {
 		return Zotero.Schema.schemaUpdatePromise;
 	});
-}
\ No newline at end of file
+}
+
+
+/**
+ * Imports an attachment from a test file.
+ * @param {string} filename - The filename to import (in data directory)
```

Comment:
No, with the hyphen is the style I've been using. It makes it easier to read from the code itself (and is what JSDoc uses in their examples).